roles/balerion.rb

```diff
@@ -21,8 +21,6 @@
   :postgresql => {
     :settings => {
       :defaults => {
-        :shared_buffers => "8GB",
-        :maintenance_work_mem => "7144MB",
```

Comment:
If we're going to drop this from the per-server configuration do we not want to add a shared setting? Otherwise it will drop to the 64Mb default?

Reply:
Looks like a rebase mistake wiped out that change

Reply:
Looks like a rebase mistake wiped out that change